packages/sights/src/build/buildJSONs.ts

```diff
@@ -67,6 +69,52 @@ function mapSights(
   );
 }
 
+function mapWireFrames(vehicles: VehicleDictionary): WireFrameDictionary {
+  const wireFrames: WireFrameDictionary = {};
+  Object.keys(vehicles).forEach((vehicle) => {
+    if (!existsSync(join(MONK_DATA_PATH, vehicle, 'partSelectionWireframes'))) return;
+    const svgFrontLeft = readOverlay(
+      join(
+        MONK_DATA_PATH,
+        vehicle,
+        'partSelectionWireframes',
+        `${vehicle}-${PartSelectionOrientation.FRONT_LEFT}.svg`,
+      ),
+    );
+    const svgFrontRight = readOverlay(
+      join(
+        MONK_DATA_PATH,
+        vehicle,
+        'partSelectionWireframes',
+        `${vehicle}-${PartSelectionOrientation.FRONT_RIGHT}.svg`,
+      ),
+    );
+    const svgRearLeft = readOverlay(
+      join(
+        MONK_DATA_PATH,
+        vehicle,
+        'partSelectionWireframes',
+        `${vehicle}-${PartSelectionOrientation.REAR_LEFT}.svg`,
+      ),
+    );
+    const svgRearRight = readOverlay(
+      join(
+        MONK_DATA_PATH,
+        vehicle,
+        'partSelectionWireframes',
+        `${vehicle}-${PartSelectionOrientation.REAR_RIGHT}.svg`,
+      ),
+    );
+    wireFrames[vehicle as VehicleModel] = {
+      [PartSelectionOrientation.FRONT_LEFT]: svgFrontLeft,
+      [PartSelectionOrientation.FRONT_RIGHT]: svgFrontRight,
+      [PartSelectionOrientation.REAR_LEFT]: svgRearLeft,
+      [PartSelectionOrientation.REAR_RIGHT]: svgRearRight,
+    };
```

Comment:
Instead of doing 4 same blocks of code like this, iterate over the orientations like this :

```javascript
wireframes[vehicle as VehicleModel] = Object.values(PartSelectionOrientation)
  .reduce((prev, curr) => ({
    ...prev,
    [curr]: readOverlay(
      join(
        MONK_DATA_PATH,
        vehicle,
       'partSelectionWireframes',
       `${vehicle}-${curr}.svg`,
      ),
    ),
  }), {} as PartSelectionWireFrames);
```